### ISSUE TITLE
User Review

### ISSUE THREAD

**Issue by edavidj** [OpenLiberty/draft-guide-operator-openshift#16]
Checklist:
- [x] Record the time it takes to go through the entire guide
- [x] Test each step in the guide is clear and easy to understand
- [x] Ensure no logic errors
- [x] Resolve anything that does not work
- [x] Resolve anything usability issues (confusing or hard to follow)
- [x] Achieve basic understanding/working knowledge of the topic in the guide

**Comments**:
- [x] Took roughly 20 minutes without counting time spent debugging
- [x] At the start for installation it refers to the main README of the operator for installation through CLI, which doesn't mention installation. I think it'd be beneficial to instead link directly to the install section of the [user guide](https://github.com/OpenLiberty/open-liberty-operator/blob/master/doc/user-guide.adoc#operator-installation), mentioning that any recent version of the operator will do.
- [x] Tried the guide on OCP 3.11 at first and the build stage failed due to `COPY --chown` not working with older docker versions. I don't think this is worth worrying about but may be useful to note or keep in mind.
- [ ] Got an image pull backoff error with message `unauthorized: authentication required` on deployment of the olapps on a fresh OCP 4.4 cluster. This was resolved after I ran the following commands (not sure which fixed it quite frankly but I assume the first command):
```sh
oc policy add-role-to-user system:image-puller kube:admin
oc policy add-role-to-group system:image-puller system:serviceaccounts:guide -n guide
```
- [x] At the very end the route endpoint didn't display the three systems until 5-10 seconds after all pods had reached the ready state. I think it may be worth noting they may need to wait a brief period of time.

**Comment by MaiHameed** [comment]
> Got an image pull backoff error with message

I got the same error when running the guide on a fresh 4.4 cluster. Deleting the olapps and redeploying them fixed it, so the provided commands won't be necessary.
